fix.txt

```diff
@@ -0,0 +1,12 @@
+# FAILED unit/examples/test_notebooks_python.py::test_sar_deep_dive_runs - papermill.exceptions.PapermillExecutionError:  
```

Comment:
Here is a trick I use: 
I usually add the errors in the text of the PR and then I go one by one fixing them. 

This also helps other people in the future to fix similar errors.